### ISSUE TITLE
Educational and Occupational Credentials should be in schema.org

### ISSUE THREAD

**Issue by philbarker** [schemaorg/schemaorg#1779]
By *educational and occupational credentials* I mean diplomas, academic degrees, certifications, qualifications, [badges](https://www.badgealliance.org/), etc., that a person can obtain by passing some test or examination of their abilities.  See also the Connecting Credentials project's [glossary of credentialing terms](http://connectingcredentials.org/common-language-work-group-glossary/).

These are already alluded to in some pending schema.org properties, for example an Occupation or JobPosting's [qualification](http://pending.schema.org/qualifications) or a Course's [educationalCredentialAwarded](http://pending.schema.org/educationalCredentialAwarded). These illustrate how educational and occupational credentials are useful for linking career aspirations with discovery of educational opportunities. There is also related work about [verifiable claims](https://www.w3.org/2017/vc/WG/) (issue #1654) that a person posses credentials (a broader class than educational and occupational credentials), which would link individuals to occupational requirements.

Examples of websites which show these educational and occupations credentials include:

 * [Board of Certified Safety Professionals](http://www.bcsp.org/Certifications)
 * [SQA (Scottish qualifications authority)](http://www.sqa.org.uk/sqa/41278.html)
 * [City and Guilds qualifications](http://www.cityandguilds.com/qualifications-and-apprenticeships#fil=uk)
 * [Microsoft Certification](https://www.microsoft.com/en-gb/learning/certification-overview.aspx)
 * [Connecting Credentials](http://connectingcredentials.org/framework-detail/)
 * [Degree Qualifications Framework](http://degreeprofile.org/read-the-dqp/dqp-cover/)
 * [EU Qualification frameworks](https://ec.europa.eu/ploteus/en/compare)
(And many others, e.g. just about every University, College and examination board)

A lot of work on describing credentials has been carried out by the [Credential Engine](http://credentialengine.org), and they are interested in linking this to schema.org. 

Given that this is a fairly complex area, it would probably best be addressed by discussion elsewhere, using this issue as a tracking issue and for coordination with the wider schema.org community. To that end, if there is sufficient support, I'll set up W3C community group.

**Comment by thadguidry** [comment]
@philbarker Just a quick barometer test , but ... how comfortable are you with the work put into the Credential Engine and meeting the broader needs ?  Really Nice, Risky, Adequate ? what's your gut tell you from what you've seen ?

**Comment by philbarker** [comment]
@thadguidry I think there are a lot of details in the Credential Engine work that are quite US-centric, however one of the things that I am sure would happen anyway in transferring that work into a schema.org context is that those details would be flattened. schema.org isn't the place for the highly nuanced details you find in specialist vocabularies.

So I am comfortable with the US -> world wide broadening. is that the type of broader need you had in mind?

Disclosure: I am receiving money from the credential engine to support this work.

**Comment by thadguidry** [comment]
@philbarker Agreed that schema.org is not the place for nuanced details.  Yes, that's the general answer I was looking for about risks with broadening the Credential Engine work into schema.org and the risks.  Yeah, I know your getting money there, and glad you are !   BTW, I really liked what I saw in the planning pages hosted separate site: http://credreg.net/

**Comment by philbarker** [comment]
@thadguidry : @stuartasutton would be better placed that I am to comment on how close to being locked down the CTDL metadata is. I know that there is an issue about having to use things like the AlignmentObject a lot which could be avoided if competencies could be described directly in schema.org, and that doing so could be a relatively simple extension of  CategoryCode / TermDefinition hence our interest in #894 #1711 #1775 But on the whole, yes, I think it's fairly locked down.

**Comment by thadguidry** [comment]
@philbarker right, that's what I hoped.  Awesome, ok thanks for now.  Good job on the email thread discussion kickoff...I'll be in the background mostly on this issue from here on out.

**Comment by philbarker** [comment]
I have proposed a W3C Community Group to take this work forward.

It now needs the support of four other people for it to happen. To support it you must have a W3C account, and then visit the [Community Groups page](https://www.w3.org/community/), and scroll down a little to the proposed groups section. Click on the button to support the Educational and Occupational Credentials in schema.org Community Group. 

If you do not have a W3C account, registering for one is free, but you do need to give an assurance that you will not be contributing an IP that cannot be only licensed under W3C's terms. This is appropriate for contributing work to schema.org.

--Phil

**Comment by liberatr** [comment]
Excited that I found this group at such an opportune time.

The company I work for produces web sites for lots of colleges and educational institutions, and I was surprised Schema.org doesn't yet have a "college degree" schema, only "course".

Interested in following the development, I've joined the group.

**Comment by stuartasutton** [comment]
@philbarker is correct that the CTDL is "fairly locked down" (stable). There are certain areas that are still unstable. A number of those are areas deal with data from external providers (e.g., U.S. Dept. of Labor) such as aggregate  employment, earnings etc.. If you want to know what those areas are, go to http://credreg.net/ctdl/terms and click the "Show Unstable" at the bottom of the page.

**Comment by philbarker** [comment]
We've had [a W3C Community Group](https://www.w3.org/community/eocred-schema/) discussing this issue for the last few months. I think that it is pretty much a point where we can contribute some terms to resolve this issue. We focused on educational and occupational credentials that are offered, as in  statements like "completion of this program of study will lead to a BSc in Computer Science"; we did not focus on the thing that proves a specific individual has a BSc in computer science. This proposal would help someone find how they could gain a qualification, not prove that they were qualified.

On the Community Group blog I have written a [summary of what we have discussed](https://www.w3.org/community/eocred-schema/2018/05/23/progress-so-far-and-the-beginning-of-the-end/) including the use cases we are trying to enable.

Our solution involves:
- Addition of a new type EducationalOccupationalCredential (as a subtype of CreativeWork, though we have a pending issue to confirm that; comments welcome)
- Addition of four properties with domain EducationalOccupationalCredential:
-- competencyRequired
-- credentialCategory
-- educationalLevel
-- recognizedBy

- Addition of EducationalOccupationalCredential to the domain of two existing properties (with changes to their definition to reflect this):
-- validFor
-- validIn

- Addition of EducationalOccupationalCredential to the range of three existing properties:
-- educationRequirements
-- educationalCredentialAwarded
-- qualifications
(these link EducationalOccupationalCredentials to JobPostings, Occupations and Courses)

There is a [draft on appspot](http://pending.eocred-1779.appspot.com/EducationalOccupationalCredential) with more details of the changes we would make, i.e. term definitions, ranges etc.

Any thoughts / comments? To my mind the most contentious aspect is where in the schema.org hierarchy EducationalOccupationalCredential fits. We see it as a subtype of CreativeWork (credentials in general are created things, they can be copyrighted), but others may have a different view.

**Comment by danbri** [comment]
see https://github.com/schemaorg/suggestions-questions-brainstorming/issues/69 for feedback from @JayHoltslander

**Comment by danbri** [comment]
Since https://schema.org/EducationalOccupationalCredential was added to Schema.org (as a Pending term), let's keep this issue open to collect implementation experience feedback. I have tagged it with a "Pending" label within Github. /cc @RichardWallis

**Comment by gmackenz** [comment]
Should we move https://schema.org/EducationalOccupationalCredential? Been about 5 years with no activity or leave it be and close this as complete.

**Comment by philbarker** [comment]
@gmackenz I think this issue can be closed. I don't foresee changes except the usual evolution you see with any class in schema.org. Would be nice if one day Ed.Oc.Cred. moved out of "new".
